core/tests/karma.conf.js

```diff
@@ -2,7 +2,8 @@ var argv = require('yargs').argv;
 var isMinificationNeeded = (argv.minify === 'True');
```

Comment:
Can we rename this to be prodMode or similar? Otherwise it seems like it's a different thing.

Reply:
Probably want to rename the "minify" arg too.